CHANGELOG.md

```diff
@@ -0,0 +1,39 @@
+# Changelog for Palaver ZNC Module
+
+## Master
```

Comment:
Master contains the changes pending to be released, once we cut a release the future version number will go here.

Master isn't actually 1.0.1, it's 1.0.1 + changes for next release, the number in README shows the last version and not what master is currently.

Make sense?